Emby.Dlna/Didl/DidlBuilder.cs

```diff
@@ -782,36 +782,33 @@ private void WriteObjectClass(XmlWriter writer, BaseItem item, StubType? stubTyp
 
         private void AddPeople(BaseItem item, XmlWriter writer)
         {
+            if (!item.SupportsPeople)
+            {
+                return;
+            }
+
             var types = new[]
             {
                 PersonType.Director,
                 PersonType.Writer,
                 PersonType.Producer,
                 PersonType.Composer,
-                "Creator"
+                "creator"
```

Comment:
I wonder why it's not a `PersonType.Creator`?..

Reply:
No idea

Reply:
Cannot we create one?